compiler/rustc_parse_format/src/lib.rs

```diff
@@ -228,13 +229,19 @@ impl<'a> Iterator for Parser<'a> {
                         Some(String(self.string(pos + 1)))
                     } else {
                         let arg = self.argument(lbrace_end);
-                        if let Some(rbrace_byte_idx) = self.must_consume('}') {
-                            let lbrace_inner_offset = self.to_span_index(pos);
-                            let rbrace_inner_offset = self.to_span_index(rbrace_byte_idx);
-                            if self.is_literal {
-                                self.arg_places.push(
-                                    lbrace_inner_offset.to(InnerOffset(rbrace_inner_offset.0 + 1)),
-                                );
+                        match self.must_consume('}') {
```

Comment:
This could have been an `if let/else`, to avoid one level of indentation, but I don't care too much either way.